plugins/plugin-chart-echarts/src/Timeseries/transformProps.ts

```diff
@@ -124,7 +130,26 @@ export default function transformProps(chartProps: ChartProps): EchartsProps {
       left: 20,
       right: 20,
     },
-    xAxis: { type: 'time' },
+    xAxis: {
+      type: 'time',
+      axisLabel: {
+        showMinLabel: !!xAxisShowMinLabel,
+        showMaxLabel: !!xAxisShowMaxLabel,
+        formatter: (value: any) => {
+          let dateFormatter;
+
+          if (xAxisFormat === smartDateFormatter.id) {
+            dateFormatter = getTimeFormatterForGranularity('PT1S');
```

Comment:
@villebro What does the granularity field means like here (https://github.com/apache-superset/superset-ui/blob/master/plugins/legacy-plugin-chart-pivot-table/src/PivotTable.js#L68-L74)?